smartcar-auth/src/main/java/com/smartcar/sdk/VehicleInfo.java

```diff
@@ -0,0 +1,130 @@
+package com.smartcar.sdk;
+
+/**
+ * A class that creates a custom VehicleInfo object, which can be used
```

Comment:
Not sure if this jsdoc is accurate. The class provides a builder interface to create `VehicleInfo` objects. Also not sold on this doc calling out the consumer use of the class. 

`VehicleInfo` seems pretty generic, if someone wants to use the `VehicleInfo` somewhere else they shouldn't have to update the docs to say "can be used for generating an authentication URL _and_ X"

Reply:
And also because I'm actually curious, why do we use the builder pattern here specifically?

Reply:
DIscussed in slack, will refactor